cmd/kaex/expand.go

```diff
@@ -51,7 +55,20 @@ func expand() error {
 		return err
 	}
 	
-	fmt.Println(input)
+	var app api.Application
+	if err = yaml.Unmarshal([]byte(input), &app); err != nil {
+		return err
+	}
+	
+	resources := make([]string, 0)
+	
+	if result, err := yaml.Marshal(api.CreateService(app)); err == nil {
+		resources = append(resources, string(result))
+	} else {
+		return err
+	}
+
+	fmt.Printf("%s", strings.Join(resources, "\n---\n"))
```

Comment:
Consider writing to an `io.Writer` instead of using `fmt.Printf`, since `fmt.Printf` always writes to `stdout` (https://golang.org/pkg/fmt/#Printf). By passing an `io.Writer` you could also write to a buffer, which is useful when testing, e.g.:

```go
package main

import (
	"bytes"
	"fmt"
	"io"
)

func main() {
	var b bytes.Buffer // A Buffer needs no initialization.
	Something(&b)
	fmt.Print(b.String())
}

func Something(w io.Writer) {
	fmt.Fprintf(w, "Hello, world!")
}
```